From 447968016399af782af9c9a82ef97181df64c46b Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Wed, 8 Aug 2018 18:49:07 -0700
Subject: [PATCH 1/8] lineage-sdk: Comment out LineageAudioService

Requires:
https://github.com/LineageOS/android_frameworks_av/commit/deebe5a37b250df375327f327c90ecbc7238a651

*) To be reverted during later ship of topic:pie-AudioFX

Change-Id: I1f65fb90e0a7609d719080fa9e48150e1227cff7
---
 ...rg_lineageos_platform_internal_LineageAudioService.cpp | 8 ++++++++
 .../lineageos/platform/internal/LineageAudioService.java  | 2 ++
 2 files changed, 10 insertions(+)

diff --git a/lineage/jni/src/org_lineageos_platform_internal_LineageAudioService.cpp b/lineage/jni/src/org_lineageos_platform_internal_LineageAudioService.cpp
index dfee005..68c1c11 100644
--- a/lineage/jni/src/org_lineageos_platform_internal_LineageAudioService.cpp
+++ b/lineage/jni/src/org_lineageos_platform_internal_LineageAudioService.cpp
@@ -27,7 +27,9 @@
 #include "android_media_AudioErrors.h"
 
 #include <media/AudioSystem.h>
+/*
 #include <media/AudioSession.h>
+*/
 
 #include <system/audio.h>
 #include <utils/threads.h>
@@ -36,6 +38,7 @@
 
 namespace android {
 
+/*
 static const char* const kClassPathName = "org/lineageos/platform/internal/LineageAudioService";
 
 static jclass gArrayListClass;
@@ -149,8 +152,12 @@ static JNINativeMethod gMethods[] = {
             (void *)org_lineageos_platform_internal_LineageAudioService_registerAudioSessionCallback},
 };
 
+*/
+
 int register_org_lineageos_platform_internal_LineageAudioService(JNIEnv *env)
 {
+    return 0;
+/*
     jclass arrayListClass = FindClassOrDie(env, "java/util/ArrayList");
     gArrayListClass = MakeGlobalRefOrDie(env, arrayListClass);
     gArrayListMethods.add = GetMethodIDOrDie(env, arrayListClass, "add", "(Ljava/lang/Object;)Z");
@@ -165,6 +172,7 @@ int register_org_lineageos_platform_internal_LineageAudioService(JNIEnv *env)
             "audioSessionCallbackFromNative", "(ILlineageos/media/AudioSessionInfo;Z)V");
 
     return RegisterMethodsOrDie(env, kClassPathName, gMethods, NELEM(gMethods));
+*/
 }
 
 } /* namespace android */
diff --git a/lineage/lib/main/java/org/lineageos/platform/internal/LineageAudioService.java b/lineage/lib/main/java/org/lineageos/platform/internal/LineageAudioService.java
index ffd439e..6af9314 100644
--- a/lineage/lib/main/java/org/lineageos/platform/internal/LineageAudioService.java
+++ b/lineage/lib/main/java/org/lineageos/platform/internal/LineageAudioService.java
@@ -73,7 +73,9 @@ public class LineageAudioService extends LineageSystemService {
     public void onBootPhase(int phase) {
         if (phase == PHASE_BOOT_COMPLETED) {
             if (NativeHelper.isNativeLibraryAvailable()) {
+                /*
                 native_registerAudioSessionCallback(true);
+                */
             }
         }
     }
-- 
2.17.1

